common/src/main/java/com/alibaba/nacos/common/notify/DefaultPublisher.java

```diff
@@ -42,21 +40,21 @@
  */
 public class DefaultPublisher extends Thread implements EventPublisher {
     
-    private static final Logger LOGGER = LoggerFactory.getLogger(NotifyCenter.class);
+    protected static final Logger LOGGER = LoggerFactory.getLogger(NotifyCenter.class);
```

Comment:
Why change it to protected? It seems that there are no subClass.

Reply:
I implement DefaultSharePublisher which is a subclass of DefaultPublisher  to separate sharePublish from defaultPublisher.